src/ui.h

```diff
@@ -49,6 +50,20 @@ struct mvwzstr {
     int sym = 0;
 };
 
+struct uilist_entry_drawn_info {
+    int text_x_start;
+    int text_x_end;
+    int y;
+    bool include_point( point p )const {
```

Comment:
```suggestion
    bool include_point( point p ) const {
```
please run your stuff through astyle before submitting.

Also, on the following if statement, please consolidate the lines.